plugin/src/main/java/com/Acrobot/Breeze/Utils/PriceUtil.java

```diff
@@ -16,6 +19,11 @@ public class PriceUtil {
     public static final char BUY_INDICATOR = 'b';
     public static final char SELL_INDICATOR = 's';
 
+    private static final Map<Character, Integer> MULTIPLIERS = Map.of(
```

Comment:
This also is Java 9 API but the project still targets Java 8 (as it's compatible down to Java 8)